.github/workflows/ci.yml

```
@@ -0,0 +1,110 @@
+        if: matrix.primary
+
+      - name: Install Playwright Browsers
+        run: pnpm --filter starmark.dev exec playwright install --with-deps
+        if: matrix.primary
+
+      - name: Run E2E tests
+        run: pnpm test:e2e
+        if: matrix.primary
+
+      - name: Upload test coverage
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)